_components/sidenav.md

```diff
@@ -38,7 +38,7 @@ lead: "Hierarchical, vertical navigation to place at the side of a page."
       <li>Indicate where a user is within the navigational hierarchy. Use the “active” state to show users which page they have navigated to.</li>
       <li>Keep the navigation links short. They can be shorter derivatives of page titles themselves.</li>
       <li>If the navigation hierarchy is too long, users may miss items at the bottom. If it’s too deep, users may miss items that require too many clicks. Usability test to find the right balance between breadth and depth.</li>
-      <li>The width of the current menu item's highlight border is set with the <code>$sidenav-current-border-width</code> variable.</li>
+      <li>The width of the current menu item‘s highlight border is set with the <code>$sidenav-current-border-width</code> variable.</li>
```

Comment:
Thanks for changing this — but I've got to change it to the other direction! `’` instead of `‘`

Reply:
`opt-shift-}`

Reply:
Is there any way to use the `smartify` filter with our html content here?

Reply:
Could it be added here: https://github.com/18F/web-design-standards-docs/blob/master/_layouts/styleguide.html#L32 ?